docker/maintenance/SMW-conceptCache.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: SMW-conceptCache
+spec:
+  schedule: "0 8 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: logger
+            image: artifactory.wikia-inc.com/sus/mediawiki-logger:latest
+          - name: php
+            image: artifactory.wikia-inc.com/sus/mediawiki-sandbox-php:0a5674e.c752111
+            env:
+             - name: SERVER_ID
+               value: "177"
+             - name: LOG_SOCKET_ONLY
+               value: "yes"
+             - name: LOG_SOCKET_ADDRESS
+               value: "tcp://localhost:9999"
+            args:
+            - php
+            - /usr/wikia/slot1/current/src/maintenance/maintenanceTaskScheduler.php
+            - --params="--quiet"
+            - --id=410,40,1318,490
```

Comment:
do we have only those wikis?

Reply:
only on these wikis the script is run now